### PR TITLE
#38 our impact section

### PR DESCRIPTION
**Pull request by samgildea**

### CODE REVIEW

File: src/components/homepage/index-styles.js [hunk 1]
```diff
@@ -58,6 +58,69 @@ export const HeroCTA = styled.div`
...
+  }
+`
+
+export const ImpactStats = styled.div`
+  margin-top: 64px;
+  margin-bottom: 48;
+
+  @media (min-width: ${dimensions.maxwidthTablet}px) {
+    display: flex;
+    // justify-content: center;
```
Comment (josephmannis) [inline src/components/homepage/index-styles.js]: Can you remove this comment please?

File: src/components/homepage/index.js [hunk 2]
```diff
@@ -76,6 +90,38 @@ export default function HomePage({ data }) {
           </FutureImage>
         </FutureSection>
 
+        <ImpactImage>
+          <img src={data.impact_image.url} />
```
Comment (josephmannis) [inline src/components/homepage/index.js]: Alt please

File: src/components/homepage/index.js [hunk 3]
```diff
@@ -76,6 +90,38 @@ export default function HomePage({ data }) {
...
+          <div>
+            <ImpactHeader>
+              <ImpactIcon>
+                <PlantIcon />
+              </ImpactIcon>
+              <H2>{data.impact_section_title}</H2>
+            </ImpactHeader>
+            <ImpactStats>
+              {data.impact_example.map((example, idx) => (
+                <ImpactExample>
```
Comment (josephmannis) [inline src/components/homepage/index.js]: Add key please

**Comment by josephmannis** [comment]
Can you please fix the checks? I'm failing to run this locally too:

```
success onPreExtractQueries - 0.001s
error There was an error in your GraphQL query:

Field "cta_button_destination" of type "PrismicAboutPageDataCta_button_destination" must have a selection of subfields. Did you mean "cta_button_destination { ... }"?
error There was an error in your GraphQL query:

Field "products_navigation_button_destination" of type "PrismicAboutPageDataProducts_navigation_button_destination" must have a selection of subfields. Did you mean "products_navigation_button_destination { ... }"?
error There was an error in your GraphQL query:

Field "contact_button_destination" of type "PrismicAboutPageDataTeam_member_informationContact_button_destination" must have a selection of subfields. Did you mean "contact_button_destination { ... }"?
failed extract queries from components - 0.503s
warning The GraphQL query in the non-page component "/Users/prince/Documents/dev/scout/powerhouse-site/src/templates/product.js" will not be run.
Exported queries are only executed for Page components. It's possible you're
trying to create pages in your gatsby-node.js and that's failing for some
reason.

If the failing component(s) is a regular component and not intended to be a page
component, you generally want to use a <StaticQuery> (https://gatsbyjs.org/docs/static-query)
instead of exporting a page query.

If you're more experienced with GraphQL, you can also export GraphQL
fragments from components and compose the fragments in the Page component
query and pass data down into the child component — https://graphql.org/learn/queries/#fragments
success write out requires - 0.002s
success run static queries - 0.010s - 1/1 102.56/s
success run page queries - 0.024s - 6/6 253.84/s
warning Browserslist: caniuse-lite is outdated. Please run:
npx browserslist@latest --update-db

Why you should do it regularly:
https://github.com/browserslist/browserslist#browsers-data-updating
⠀
You can now view powerhouse-site in the browser.
⠀
  http://localhost:8000/
⠀
View GraphiQL, an in-browser IDE, to explore your site's data and schema
⠀
  http://localhost:8000/___graphql
⠀
Note that the development build is not optimized.
To create a production build, use gatsby build
⠀
success Building development bundle - 5.642s
error ENOENT: no such file or directory, open '/Users/prince/Documents/dev/scout/powerhouse-site/.cache/json/_about_.json'
```

**Comment by samgildea** [comment]
Sorry about that. Should work now, you may have to run gatsby clean before.

**Review by josephmannis — COMMENTED** [review]
This code looks good, but I think there's a problem with the graphql queries:
```
String cannot represent value: [{ type: "paragraph", text: "But here is an artist. He desires to paint you the dreamiest, shadiest, quietest, most enchanting bit of romantic landscape in all the valley of the Saco. What is the chief element he employs? There stand his trees, each with a hollow trunk, as if a hermit and a crucifix were within; and here sleeps his meadow, and there sleep his cattle; and up from yonder cottage goes a sleepy smoke.", spans: [] }, { type: "paragraph", text: "Deep into distant woodlands winds a mazy way, reaching to overlapping spurs of mountains bathed in their hill-side blue. But though the picture lies thus tranced, and though this pine-tree shakes down its sighs like leaves upon this shepherd’s head, yet all were vain, unless the shepherd’s eye were fixed upon the magic stream before him.", spans: [] }, { type: "paragraph", text: "Go visit the Prairies in June, when for scores on scores of miles you wade knee-deep among Tiger-lilies—what is the one charm wanting?—Water—there is not a drop of water there! Were Niagara but a cataract of sand, would you travel your thousand miles to see it? ", spans: [] }, { type: "paragraph", text: "", spans: [] }, { type: "paragraph", text: "", spans: [] }]

  14 |           }
  15 |           reasons_main_heading
  16 |           preview_description
  17 |           overview_image_two {
  18 |             url
  19 |           }
  20 |           overview_image_one {
  21 |             url
  22 |           }
  23 |           overview_heading
> 24 |           overview_description
     |           ^
  25 |           learn_more_button_text
  26 |           learn_more_button_destination {
  27 |             url
  28 |           }
  29 |           demo_video {
  30 |             url
  31 |           }
  32 |           demo_heading
  33 |           cta_button_text
  34 |           cta_button_destination {

File path: /Users/prince/Documents/dev/scout/powerhouse-site/src/templates/use_case.js
Url path: /for-your-home
```

This is probably because I let this sit, my apologies – please feel free to fix and merge.

**Pull request merged**